os/cluster-architectures.md

```diff
@@ -60,22 +49,44 @@ coreos:
         ExecStart=/usr/bin/systemctl enable docker-tcp.socket
 ```
 
+This file is used to provision your local CoreOS machine on its first boot. This sets up and enables the docker API, which is how you can use docker on your laptop. The Docker CLI manages containers running within the VM, *not* on your personal operating system.
+
+Using the Container Linux Config Transpiler, or `ct`, ([download][ct-getting-started]) convert the above yaml into an [Ignition][ignition-getting-started]. Alternatively, copy the content of the Igntion tab in the above example. Once you have the Ignition configuration file, pass it to your provider ([complete list of supported Ignition platforms][ignition-supported]).
+
+Once the local VM is running, tell your Docker binary on your personal operating system to use the remote port by exporting an environment variable and start running Docker commands. Run these commands in a terminal *on your local operating system (MacOS or Linux), not in the Container Linux virtual machine*:
+
+```sh
+$ export DOCKER_HOST=tcp://localhost:2375
+$ docker ps
+```
+
+This avoids discrepencies between your development and production environemnts.
+
+### Related local installation tools
+
+There are several different options for testing Container Linux locally:
+
+- [coreos-vagrant][coreos-vagrant] is not an officially fully supported platform, however may be a good resource to check out if you are already comfortable with Vagrant and are unable to use one of Ignition's officially supported platforms.
+- [Minikube][minikube] is used for local Kubernetes development. This does not use CoreOS but is very fast to setup and is the easiest way to test-drive use Kubernetes.
+
 ## Small cluster
```

Comment:
the idea is that you can run etcd + any other app, but not run a dedicated etcd cluster that would put you over budget